keyboards/gmmk/pro/rev1/ansi/config.h

```diff
@@ -21,3 +21,9 @@
 #define DRIVER_1_LED_TOTAL 66
 #define DRIVER_2_LED_TOTAL 32
 #define RGB_MATRIX_LED_COUNT (DRIVER_1_LED_TOTAL + DRIVER_2_LED_TOTAL)
+
+// Timeout lights after 5m or sleep/usb suspended
+#define RGBLIGHT_SLEEP
+#define RGB_DISABLE_TIMEOUT 300000
```

Comment:
`RGB_DISABLE_TIMEOUT` isnt an option on develop
```suggestion
#define RGB_MATRIX_TIMEOUT 300000
```